README.md

```diff
@@ -10,17 +10,16 @@ ember install ember-cli-lazysizes
 
 ## Usage
 
-Usage is simple, all you have to do is to assign the class (default: `'lazyload'`) to image tag in your template:
+Usage is simple, all you have to do is to assign the class (default: `'lazyload'`) to any image tag in your template,
+and use `data-src` or `data-srcset` attributes instead of `src` or `srcset`:
 
 ```html
-<img src="./tomster.png" alt="tomster" class="lazyload">
+<img data-src="tomster.png" alt="tomster" class="lazyload">
 ```
 
-and the response is the custom generated class (default: `'lazyloaded'`):
+See the lazysizes' [markup API](https://github.com/aFarkas/lazysizes#markup-api) for all the supported loading patterns.
 
-```html
-<img src="./tomster.png" alt="tomster" class=" lazyloaded">
```

Comment:
I think it's not imported to show how lazysizes.js will alter the DOM, might be confusing if users think *they* have to do it!